packages/msw-addon/src/mswDecorator.js

```diff
@@ -1,6 +1,11 @@
 let api
+let IS_BROWSER
 
-const IS_BROWSER = typeof global.process === 'undefined'
+try {
+  IS_BROWSER = typeof global.process === 'undefined'
```

Comment:
Thing is, that relying on the `global` is not a good idea to begin with. There's more to determining whether the process runs in a browser or not, especially given that some environments emulate browser-like global objects, but still run in Node.js.

I'd recommend using the [is-node-process](https://github.com/mswjs/is-node-process) package from our ecosystem to determine whether this add-on executes in the browser:

```js
import { isNodeProcess } from 'is-node-process'

// Any non-Node.js process is a browser process.
const IS_BROWSER = !isNodeProcess()
```

That should cover the add-ons requirements and fix your issue because the said package doesn't rely on `global`.

@yannbf, please, Yann, what do you think about this?

Reply:
Hey @kettanaito sorry for taking long to reply. Even though the logic is rather simple, I think we should go with the package! It's only 267b gzipped 🤷

Reply:
IMO package size doesn't really matter for devDeps